AmericanMedicalAssociation/AMA-style-guide
# Move docs up a level and fix links.

## Ticket(s)

Please do not submit a Pull Request without a relevant ticket! If you are creating a new pattern or feature, create an issue describing the need for this feature Github **first** and then link to it below.

**Github Issue**

na

**Jira Ticket**

na

## Description

As per Slack discussion, move docs up a level and fix links.

## To Test

- Observe the docs folder at the root of the repo.
- Click links to docs in updated files and make sure they aren't broken.


## Relevant Screenshots/GIFs

Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks

Remaining tasks?


## Additional Notes

Anything more to add?
